composer.json

```diff
@@ -5,32 +5,38 @@
   "license": "GPL-2.0-or-later",
   "require": {
     "php": ">=5.6",
-    "composer/installers": "^2.2"
+    "composer/installers": "^v1.12.0 || ^2.2",
+    "dealerdirect/phpcodesniffer-composer-installer": "^1.0.0",
+    "squizlabs/php_codesniffer": "^3.7",
```

Comment:
Could we remove this as it would be loaded with `wp-coding-standards/wpcs`?

Reply:
Sure yeah that works.

I figured it's best to keep the explicit reference here like we do today, but right now the result is the same.